docs/usage/configuration/config_documentation.md

```diff
@@ -3702,8 +3753,9 @@ federation_sender_instances:
 ---
 ### `instance_map`
 
-When using workers this should be a map from worker name to the
+When using workers this should be a map from [`worker_name`](#worker_name) to the
 HTTP replication listener of the worker, if configured.
+Not every type of worker needs a HTTP replications listener.
```

Comment:
Can we say _which_ workers need a replication listener here? Is this any worker which accepts requests from other workers? E.g. event persister, pushers? Maybe we should refer to `stream_writers`?

Is this (should this be) explained in the worker docs page?

Reply:
See: https://github.com/matrix-org/synapse/pull/14086/commits/17ec5d3cf1a6398480647753ed466f955d2466ee